src/main/java/frc/robot/subsystems/scoring/AimerIORoboRio.java

```diff
@@ -0,0 +1,179 @@
+
+        TalonFXConfigurator aimerRightConfig = aimerRight.getConfigurator();
+        aimerRightConfig.apply(
+                new CurrentLimitsConfigs()
+                        .withStatorCurrentLimit(120)
+                        .withStatorCurrentLimitEnable(true));
+
+        aimerRight.setPosition(0.0);
+    }
+
+    @Override
+    public void setAimAngleRad(double goalAngleRad, boolean newProfile) {
+        this.goalAngleRad = goalAngleRad;
+        this.newProfile = newProfile;
+    }
+
+    @Override
+    public void controlAimAngleRad() {
+        if (goalAngleRad != previousGoalAngle && newProfile) {
+            timer.reset();
+            timer.start();
+
+            initialAngle = getEncoderPosition();
+            initialVelocity = velocity;
+
+            previousGoalAngle = goalAngleRad;
+        }
+        goalAngleRad = MathUtil.clamp(goalAngleRad, minAngleClamp, maxAngleClamp);
+    }
+
+    @Override
+    public void setAngleClampsRad(double minAngleClamp, double maxAngleClamp) {
+        this.minAngleClamp = minAngleClamp;
+        this.maxAngleClamp = maxAngleClamp;
+    }
+
+    @Override
+    public void setOverrideMode(boolean override) {
+        this.override = override;
+    }
+
+    @Override
+    public void setOverrideVolts(double volts) {
+        overrideVolts = volts;
+    }
+
+    @Override
+    public void setPID(double p, double i, double d) {
+        controller.setP(p);
+        controller.setI(i);
+        controller.setD(d);
+    }
+
+    @Override
+    public void setMaxProfile(double maxVelocity, double maxAcceleration) {
+        profile =
+                new TrapezoidProfile(
+                        new TrapezoidProfile.Constraints(maxVelocity, maxAcceleration));
+    }
+
+    @Override
+    public void setFF(double kS, double kV, double kA, double kG) {
+        feedforward = new ArmFeedforward(kS, kG, kV, kA);
+    }
+
+    private double getEncoderPosition() {
+        // return encoder.getAbsolutePosition() * 2.0 * Math.PI -
+        // ScoringConstants.aimerEncoderOffset;
+        return aimerRight.getPosition().getValueAsDouble() * (2.0 * Math.PI) / 80.0;
+    }
+
+    @Override
+    public void updateInputs(AimerIOInputs inputs) {
+        State trapezoidSetpoint =
+                profile.calculate(
+                        timer.get(),
+                        new State(initialAngle, initialVelocity),
+                        new State(goalAngleRad, 0));
+
+        if (override) {
+            appliedVolts = overrideVolts;
+        } else {
+            appliedVolts =
+                    feedforward.calculate(trapezoidSetpoint.position, trapezoidSetpoint.velocity)
+                            + controller.calculate(
+                                    getEncoderPosition(), trapezoidSetpoint.position);
+        }
+
+        if (getEncoderPosition() > Math.PI / 2.0) {
+            appliedVolts = 0.0;
+        }
+        aimerRight.setVoltage(appliedVolts);
```

Comment:
If you comment this line out and enable teleop, then the aimer shouldn't be able to move and we would be able to plot in scope variables that help determine the root cause

Reply:
Maybe we should also clamp appliedVolts between -12 and 12 for sanity.